README.md

```diff
@@ -70,10 +70,23 @@ The project-specific configuration file [gradle.properties][file:gradle.properti
 | `pluginUntilBuild`        | The `until-build` attribute of the <idea-version> tag.                                     |
 | `platformType`            | The type of IDE distribution.                                                              |
 | `platformVersion`         | The version of the IntelliJ Platform IDE that will be used to build the plugin.            |
-| `platformDownloadSources` | IDE sources downloaded while initializing the Gradle build.                                    |
+| `platformDownloadSources` | IDE sources downloaded while initializing the Gradle build.                                |
 
 The properties listed define the plugin itself or configure the [gradle-intellij-plugin][gh:gradle-intellij-plugin] – check its documentation for more details.
 
+### Dependency on the Kotlin standard library
+
+Since Kotlin 1.4, a dependency on a standard library (`stdlib`) is added automatically.
+In most cases, it is not necessary to attach such libraries within the bundled plugin.
+ 
+The [gradle.properties][file:gradle.properties] file explicitly alters the default behaviour of the Kotlin plugin using the opt-out property:
```

Comment:
..default behaviour of the Kotlin Gradle plugin by specifying this opt-out property: